docs/advanced_topics/streamfield_migrations.md

```diff
@@ -101,45 +115,55 @@ def pagerevision_to_streamfield(revision_data):
 
 
 def page_to_richtext(page):
-    changed = False
-    if page.body.raw_text is None:
-        raw_text = ''.join([
-            child.value.source for child in page.body
-            if child.block_type == 'rich_text'
-        ])
-        page.body = raw_text
-        changed = True
+    if page.body:
```

Comment:
I think this is missing the `changed = False` initialisation?

Reply:
Ouch, good spot! Fixed in dcded815f63a4f456bb5d8fd8874b55944b27ceb.